src/translate.tsx

```diff
@@ -1,15 +1,19 @@
-import { marked } from 'marked'
 import React from 'react'
+import { marked } from 'marked'
 
 const defaultOptions = {
   markdown: false,
 }
 
 export default (singleton) => {
-  return function translate(text, plural, options) {
+  return function translate(
+    text: string,
+    plural?: string | { [key: string]: any },
+    options?: { [key: string]: any }
```

Comment:
This is a really weird construct as the second option can be omitted, but you could still provide the 3rd one instead.
Hence you have some weird typing allowances here ... 🤔

Reply:
Should we maybe reconsider the API to make it simpler?
We also had some confusion with it, while adding declarations to workflow-client

Reply:
We could potentially think of providing an alternative API entry point. 

If we really want to modify this interface, then we might have to consider writing codemod for transforming previous calls in dependent project. (depending on the changes proposed) 

First of all I actually wanted to ensure a bit of stability for this project as it hasn't been touch for a long time.